.eslintrc.json

```diff
@@ -42,11 +42,11 @@
         "props": false
       }
     ],
-    "complexity": [ 2, 3 ],
-    "max-depth": [2, 3],
-    "max-len": [1, 160, 4],
+    "complexity": [ 2, 10 ],
```

Comment:
Ok I already created some issues in order to fix this since at the moment most of the code on this branch is too complex, but yes I agree that is kind of risky to do some changes we need a suite of tests in order to make sure we add changes that does not break things.